lib/config/testdata/v18.xml

```diff
@@ -0,0 +1,15 @@
+<configuration version="17">
```

Comment:
Ok. I was simply comparing to v17.xml where the numbers matched, no "deeper insights" :)